lib/shippingeasy_integration/server.rb

```diff
@@ -48,7 +50,7 @@ class Server < EndpointBase::Sinatra::Base
         order =
           ShippingEasy::Resources::Order
           .find(
-            id: @payload[:shipping_easy][:order][:sync_id]
+            id: @config[:sync_id]
```

Comment:
@misteral can you explain what `@config` is or what values are in that hash?  Without me knowing anything, it would seem that `@config` is more for the settings that would be used in every request such as the `store_api_key` and not so much for params that would be different request to request such as the `sync_id`.

Reply:
@config hash of values from parameters json. (parameters: @config)

Reply:
https://github.com/Sweetist/endpoint_base/blob/master/lib/endpoint_base/concerns/param_processor.rb#L49